pkgs/development/libraries/aubio/default.nix

```diff
@@ -18,6 +18,13 @@ stdenv.mkDerivation rec {
   dontAddWafCrossFlags = true;
   wafFlags = lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) "--disable-tests";
 
+  postPatch = ''
+    # U was removed in python 3.11 because it had no effect
+    substituteInPlace waflib/*.py \
+      --replace "m='rU" "m='r" \
+      --replace "'rU'" "'r'"
+  '';
```

Comment:
@AndersonTorres @vrthra There's failures related to this in many waf packages because they vendor waflib

Reply:
aubio in github is a bit different:

https://github.com/aubio/aubio